src/gt4py/next/otf/compiled_program.py

```diff
@@ -229,13 +229,20 @@ class CompiledProgramsPool:
         stages.CompiledProgram | concurrent.futures.Future[stages.CompiledProgram],
     ] = dataclasses.field(default_factory=dict, init=False)
 
+    @functools.cached_property
+    def _numeric_values_extractor(self) -> Callable | None:
```

Comment:
```suggestion
    def _primitive_values_extractor(self) -> Callable | None:
```
Or am I missing something?